museum/sampleRecords/locationHistory.destination.json

```diff
@@ -0,0 +1,245 @@
+{
+  "ok":true,
+  "results":[
+    {
+      "movement_id":"1",
+      "id":"1",
+      "idno":"1",
```

Comment:
I did think about that, and yes they probably don't need an idno, but it kind of makes sense to proxy the legacy id's here as there is nothing the the movement that can serve as an `idno`, and it means that we can potentially re-import based on `idno` and just overwrite existing records without having to jump through hoops linking up using the `legacyId` field. This is different from Subject List entries as those already have a putative idno in the form of the subject name which is unique in the source data.